### ISSUE TITLE
Import Wikia sites into Xowa Android App

### ISSUE THREAD

**Issue by PhongTong** [gnosygnu/xowa#202]
How can I import wikis from Wikia farm? Can I do it like PC version? Thanks :)

**Comment by gnosygnu** [comment]
Unfortunately, there isn't an easy way. You'd have to generate an HTML version of the XOWA database. Right now, the only way is through a command-line script.

If you want to give it a try, you can look at http://xowa.org/home/wiki/Dev/Command-line/Dumps and try it for simple.wikipedia.org. Once that works, try it for whatever wikia you're interested in.

Let me know if you need more specific info. Thanks.

**Comment by PhongTong** [comment]
So I need to replace all the "commons.wikimedia.org" and "simple.wikipedia.org" text to wikia site URL and paste it into CMD? For example, this wiki: http://sonako.wikia.com

By the way, I have troubled when trying import xml files I get from wikia sites on another PC? 

It only shows "preparing import" in Show Process Manager popup but nothing happens

Operation: Win 7 64 bit. Xowa version: 4.5.19.1801

**Comment by gnosygnu** [comment]
Yeah, I forgot to mention this is a known bug. See #199. You can download the patch I made in this link: https://drive.google.com/open?id=1CxodcsqNcrnYawydePUMGkhWonAHwBBs

Also, you should look at #201. I think Wikia completely changed their Infobox format and XOWA doesn't know how to handle it.

Thanks.
